spyder/plugins/variableexplorer/widgets/namespacebrowser.py

```diff
@@ -73,6 +73,13 @@ def __init__(self, parent):
         self.editor = None
         self.shellwidget = None
         self.finder = None
+        self.pane_empty = PaneEmptyWidget(
+            self,
+            "variable-explorer",
+            _("You haven't defined any variables yet."),
+            _("Run code with variables in the Editor or IPython console to "
+              "see them listed here for exploration and editing.")
```

Comment:
```suggestion
            _("Run code in the Editor or IPython console to see any "
              "top-level variables listed here for exploration and editing.")
```

All code has variables; this clarifies that we're specifically talking about _top-level_ variables here. We could also say "global variables", but that's likely to confuse many users as they might not be familiar with the term, or might think it applies only to when using the `global` statement, while "top-level" is more intuitive and directly connected with the nature of the code that was actually run.

Reply:
I'm not so sure about `top-level variables`. For instance, how that would be translated to Spanish? `Variables de primer nivel` sounds really odd. But `variables globales` (for `global variables`) sounds simple and direct.

Reply:
Then maybe just translate `top-level variables` to Spanish as `variables globales` then, as the most appropriate term in that language? As mentioned, its likely to be potentially confusing in English, since the word and phrasing is exactly the same as the `global` keyword, whereas top-level variables is more clear and descriptive.

Reply:
Isn't the term "global variables" exactly what is meant here? It is what the Python tutorial uses, right after introducing functions: https://docs.python.org/3/tutorial/controlflow.html#defining-functions . It is what I expect every course to use when they explain functions and that there is a difference between variables inside and outside a function, even before mentioning that there is a `global` statement.

I think having a non-standard term is more confusing. People who know the term "global variables" will wonder what is meant with "top-level variables". And I am not sure it is clearer; is the global scope the bottom level or the top level?

Reply:
Okay, makes sense then :+1:

Reply:
Ok, then I'm going to leave the right suggestion for @dalthviz to apply it:

```suggestion
            _("Run code in the Editor or IPython console to see any "
              "global variables listed here for exploration and editing.")
```

Reply:
Oh oops—uncharacteristically forgot to actually update the suggestion! Thanks for that.